tests/convergence/BAE.cpp

```diff
@@ -0,0 +1,383 @@
+//
```

Comment:
This needs the standard legal mumbo jumbo at the top

Reply:
done